### PR TITLE
remove wallet selectors

### PR DESCRIPTION
**Pull request by wolverineks**

### CODE REVIEW

File: src/yoroi-wallets/cardano/types.ts [hunk 1]
```diff
@@ -336,4 +343,11 @@ const yoroiWalletKeys: Array<YoroiWalletKeys> = [
   'toJSON',
   'fetchCurrentPrice',
   'fetchFundInfo',
+  'internalAddresses',
+  'externalAddresses',
+  'confirmationCounts',
```
Comment (stackchain) [inline src/yoroi-wallets/cardano/types.ts:348]: `confirmationCounts` is not in use anymore (calculated on tx details)

File: src/AppNavigator.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ import {WalletInitNavigator} from './WalletInit/WalletInitNavigator'
 import {WalletNavigator} from './WalletNavigator'
 import {walletManager} from './yoroi-wallets'
 
-const IS_STORYBOOK = env.getBoolean('IS_STORYBOOK', false)
+const IS_STORYBOOK = !env.getBoolean('IS_STORYBOOK', false)
```
Comment (stackchain) [inline src/AppNavigator.tsx]: :D 
```suggestion
const IS_STORYBOOK = env.getBoolean('IS_STORYBOOK', false)
```

**Review by stackchain — COMMENTED** [review]
perhaps the title/branch name are busted

**Comment by wolverineks** [comment]
> perhaps the title/branch name are busted

what?

**Comment by stackchain** [comment]
> > perhaps the title/branch name are busted
> 
> what?

this comment should be on -> #2144 (can we close it?)

File: src/yoroi-wallets/cardano/types.ts (hunk 1 above)
Comment (wolverineks) [reply src/yoroi-wallets/cardano/types.ts:348]: its used to derive `TransactionInfo`

**Review by stackchain — APPROVED** [review]

**Pull request merged**